qemu/tests/cfg/block_commit.cfg

```diff
@@ -27,3 +27,11 @@
                     wait_timeout = 120
                     before_start = "load_stress"
                     after_finished = "unload_stress verify_backingfile reboot verify_alive"
+                - with_stop_cont:
+                    before_start = "stop"
+                    after_finished = "resume reboot verify_alive"
```

Comment:
@xiywang I hit login timeout error when test this case, please add verify guest alive step before stop it, maybe can avoid the error and more fit the test scenario. Thanks.